backend/app/routers/public_datasets.py

```diff
@@ -0,0 +1,353 @@
+import datetime
```

Comment:
Just curious is there any similarity to `datasets.py`?

Reply:
it has some of the methods of datasets but not the Depends(get_current_user) and it does not have a dependency added to the router in main.

It only has the get methods.